src/App.js

```diff
@@ -62,7 +62,7 @@ function App() {
           <div>
             <label
               htmlFor="email"
-              className="block text-base font-medium leading-6 text-white"
+              className="block text-base font-medium leading-6 text-white font-IBMPlexSansRegular"
```

Comment:
Can we have a better name for this? Usually you don't need that many fonts, so this could be:

```suggestion
              className="block text-base font-medium leading-6 text-white font-sans"
```

If you have multiple `sans` fonts, first of all, why? Secondly, also why, but why do you really need it and maybe name it that, like:

```suggestion
              className="block text-base font-medium leading-6 text-white font-sans-accent"
```